lib/file_manipulation.rb

```diff
@@ -0,0 +1,23 @@
+class FileManipulation
+  
+  def initialize(path_to_file)
+    @path_to_file = path_to_file
+    File.open(path_to_file)
```

Comment:
Aqui vc n está atribuindo a instancia do file pra nenhuma variavel
```suggestion
    @file = File.open(path_to_file)
```